### PR TITLE
Add an option to public-ids to pass the principal.

### PR DESCRIPTION
**Pull request by jplevyak**
This can be used to verify the account_id of a principal.

Signed-off-by: John Plevyak <jplevyak@gmail.com>

### CODE REVIEW

File: tests/outputs/public-ids.txt [hunk 1]
```diff
@@ -1,2 +1,4 @@
 Principal id: fdsgv-62ihb-nbiqv-xgic5-iefsv-3cscz-tmbzv-63qd5-vh43v-dqfrt-pae
 Account id: 345f723e9e619934daac6ae0f4be13a7b0ba57d6a608e511a00fd0ded5866752
+Principal id: 44mwt-bq3um-tqicz-bwhad-iipx4-6wzex-olvaj-z63bj-wkelv-xoua3-rqe
```
Comment (chmllr) [inline tests/outputs/public-ids.txt:3]: We usually add new tests for new sets of call options to make sure we didn't introduce regressions.

Also I wonder if we still should require PEM for that command and print the public ids derived from it too? It feels like we're mixing concerns here. Maybe we we can make it an either/or functionality or add a completely new account-id lookup command, WDYT?

Comment (jplevyak) [reply tests/outputs/public-ids.txt:3]: I did add a test for the new call option and this is it.  If found this useful, but if you want to do it some other way or in another tool, you tell me.  The public-ids command seemed to be the most relevant to me.

Comment (chmllr) [reply tests/outputs/public-ids.txt:3]: Oh I'm sorry I overlooked that you actually _added_ a new command to the same file, and now I understand that the command behaves exactly as I'd expect it (I assumed initially that the PEM ids are printed too, but this output is from the call above the new one) So all good then, thanks a lot John!

**Review by chmllr — APPROVED** [review]

**Comment by jplevyak** [comment]
bors merge

**Comment by jplevyak** [comment]
@chmllr I don't have permission to merge.

**Pull request merged**